definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -19867,6 +19867,48 @@
     "UserRole": "citizen",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "23/07/2020",
```

Comment:
`LiveFrom` is ignored by CCD, but it helps a little bit to investigate what happened when (even though we can get more reliable info from git blame).

Can you please update all `LiveFrom` with current date?

Reply:
:white_check_mark: